pkg/platform/github.go

```diff
@@ -76,3 +88,71 @@ func NewGitHub(ctx context.Context, cfg *gh.Config) (*GitHub, error) {
 
 	return g, nil
 }
+
+// RequestReviewers abstracts GitHub's RequestReviewers API with retries.
+func (g *GitHub) requestReviewers(ctx context.Context, reviewer *github.ReviewersRequest) error {
```

Comment:
The mix of singular and plural is a little confusing, should `reviewer` be `reviewers`?

Reply:
Renamed to `reviewers`. Did the same with `inputs` to `input`, matching the singular `AssignReviewersInput`.